debezium-core/src/main/java/io/debezium/relational/topic/TopicMapper.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.relational.topic;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.SchemaBuilder;
+
+import io.debezium.relational.Table;
+
+import java.util.Map;
+
+/**
+ * @author David Leibovic
+ */
+public abstract class TopicMapper {
+
+    /**
+     * Get the name of the topic given for the table.
+     *
+     * @param topicPrefix prefix for the topic
+     * @param table the table that we are getting the topic name for
+     * @return the topic name; may be null if this strategy could not be applied
+     */
+    abstract public String getTopicName(String topicPrefix, Table table);
```

Comment:
I'm not sure we need to enforce this, because at some point I do think at some point that we might be able to support a topic that has disparate tables. I do think that we need better logging of what the mapping is actually doing and when it kicks in.